### PR TITLE
Activity.package

### PR DESCRIPTION
**Pull request by bgerber123**
Look at prob.overlap...

test code...

y=c(6.25753965, 0.56420439, 0.17951958, 0.97453486, 5.64204395, 6.23189400, 0.28210220, 5.92414615, 0.92324356, 5.71898091, 0.43597612,
    5.43687871, 0.66678701, 0.07693696, 5.59075264, 0.38468481, 0.74372398, 5.18042217, 0.35903916, 0.46162178, 1.07711748, 0.84630659,
    5.25735913, 1.12840879, 1.25663706, 5.20606783, 5.87285484, 0.00000000, 0.12822827, 5.94979180, 5.53946133, 0.53855874, 6.20624834)

library(activity)
activity::density2

mod1<-fitact(y) 
plot(mod1)
probs=prob.overlap(densityplot=mod1,dawn=c(6,7),dusk=c(17,18))
posthoc.niche(probs,hyp.sets("Traditional"))

### CODE REVIEW

File: R/prob.Overlap.r [hunk 1]
```diff
@@ -11,13 +11,16 @@
 #' information.
 #' @return A matrix of three probabilities.
 #' @details
-#' When creating the density plot, it is important to increase the \code{n.grid}
+#' When creating the density plot from overlap, it is important to increase the \code{n.grid}
```
Comment (mfidino) [inline R/prob.Overlap.r:14]: Looks like this has overlap instead of activity written in the text?

File: DESCRIPTION [hunk 2]
```diff
@@ -46,6 +46,7 @@ Suggests:
     ggplot2 (>= 3.3.6),
     testthat (>= 3.0.0),
     overlap  (>= 0.3.4),
```
Comment (mfidino) [inline DESCRIPTION:48]: Does this need to get removed as well if it's off of CRAN?

File: R/prob.Overlap.r (hunk 1 above)
Comment (bgerber123) [reply R/prob.Overlap.r:14]: I left the overlap stuff in there. So, can use activity or overlap

File: DESCRIPTION (hunk 2 above)
Comment (bgerber123) [reply DESCRIPTION:48]: It doesn't really do anything. Can still download old versions. Not sure what people are going to do.

**Pull request merged**